web-bundle/src/main/java/com/graphhopper/http/PtRedirectFilter.java

```diff
@@ -40,6 +40,13 @@ public void filter(ContainerRequestContext requestContext) {
 
     private boolean shouldRedirect(ContainerRequestContext requestContext) {
         String maybeVehicle = requestContext.getUriInfo().getQueryParameters().getFirst("vehicle");
-        return maybeVehicle != null && maybeVehicle.equals("pt");
+        if (maybeVehicle != null && maybeVehicle.equals("pt")) {
+            return true;
+        }
+        String maybeProfile = requestContext.getUriInfo().getQueryParameters().getFirst("profile");
+        if (maybeProfile != null && maybeProfile.equals("pt")) {
+            return true;
+        }
+        return false;
```

Comment:
this could be a bit shorter ;)

`return "pt".equals(maybeProfile)`

And should we also remove the profile param in the filter method above? (`replaceQueryParam("profile")`)

Reply:
> this could be a bit shorter ;) return "pt".equals(maybeProfile)

Ah yes I was trying to keep the symmetry :)

> And should we also remove the profile param in the filter method above? 

Good idea, yes, I think so.